templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -652,8 +652,12 @@ contents:
     }
 
     # Accepts parameters $iface, $iface_default_hint_file, $ip_hint_file
-    # Finds the nodeip interface from the interface that matches the ip address in $ip_hint_file.
-    # Otherwise fallbacks to a previously used interface or to the default interface.ç
+    # Determines the interface to be used for br-ex. Order of priority is:
+    # 1. Use the user specified interface if provided in the default bridge file
+    # 2. Use the node IP hint interface
+    # 3. Use the interface detected as default gateway
+    # nodeip interface from the interface that matches the ip address in $ip_hint_file.
+    # Otherwise fallbacks to a previously used interface or to the default interface.
```

Comment:
Couple of nits

I would rework the full comment of this function:
```
    # Accepts parameters $iface_default_hint_file, $extra_bridge_file, $ip_hint_file
    # Determines the interface to be used for br-ex. Order of priority is:
    # 1. Use the user specified interface if provided in the default bridge file
    # 2. Use the node IP hint interface
    # 3. Use the previously selected interface
    # 4. Use the interface detected as default gateway
    #
    # Read $default_bridge_file and return contained the interface. Otherwise,
    # read $ip_hint_file and return the interface that matches this ip. Otherwise,
    # if the default interface is br-ex, use that and return.
    # If the default interface is not br-ex:
    # Check if there is a valid hint inside $iface_default_hint_file. If so, use that hint.
    # If there is no valid hint, use the default interface that we found during the step
    # earlier. 
    # Never use the interface that is provided inside $extra_bridge_file for br-ex1.
    # Never use br-ex1.
    # Write the default interface to $iface_default_hint_file
```

Also consider passing $default_bridge_file in. Not an issue, just commonality with how $ip_hint_file is passed in as well.